lib/PEX.ts

```diff
@@ -525,7 +530,8 @@ export class PEX {
 
     let presentation = presentationResult.presentation;
 
-    if (CredentialMapper.isSdJwtDecodedCredential(presentationResult.presentation)) {
+    if (CredentialMapper.isSdJwtDecodedCredential(presentationResult.presentation as unknown as SdJwtDecodedVerifiableCredential)) {
+      // FIXME? SdJwtDecodedVerifiableCredentialWithKbJwtInput is local type and is not supported in ssi-sdk
```

Comment:
It has a FIXME in there, but this is not a new situation. We still pass the same data into isSdJwtDecodedCredential, typescript just got in the way suddenly.

Reply:
That comment makes no sense IMO.  Obviously the types changed

Reply:
SdJwtDecodedVerifiableCredentialWithKbJwtInput does not overlap with SdJwtDecodedVerifiableCredential
Was is not matching is local type 
```
export interface SdJwtKbJwtInput {
  header: {
    typ: 'kb+jwt';
  };
  payload: {
    iat: number;
    sd_hash: string;
    nonce?: string;
  };
}
```

vs in SSI-SDK
```
  kbJwt?: {
    compact: CompactJWT
    payload: SdJwtVcKbJwtPayload
  }
  ```

Reply:
We have the latest SSI types now, so perhaps PresentationResult should be like this now
```
export interface PresentationResult {
  /**
   * The resulting presentation, can have an embedded submission data depending on the location parameter
   */
  presentation: OriginalVerifiablePresentation;
  ```

Reply:
I synched it up with ssi-types, but since some of the fields in header & payload have to be loaded by the signer I had to change the types a bit
```
export type PartialSdJwtKbJwt = {
  header: Partial<SdJwtVcKbJwtHeader>;
  payload: Partial<SdJwtVcKbJwtPayload>;
};

export type PartialSdJwtDecodedVerifiableCredential = Omit<SdJwtDecodedVerifiableCredential, 'kbJwt'> & { kbJwt: PartialSdJwtKbJwt };
```